utils/loggingcpp/format.h

```diff
@@ -0,0 +1,94 @@
+/*
+   Copyright (C) 2014 InfiniDB, Inc.
```

Comment:
I doubt that InfiniDB adds something to this file except maybe a header.

Reply:
:)